### PR TITLE
Lower low level for many messages to improve UX. Improve wording of some messages.

### PR DESCRIPTION
**Pull request by danlaine**

### CODE REVIEW

File: vms/platformvm/common_blocks.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ func (cb *CommonBlock) parentBlock() Block {
 	parentID := cb.ParentID()
 	parent, err := cb.vm.getBlock(parentID)
 	if err != nil {
-		cb.vm.Ctx.Log.Debug("could not get parent (ID %s) of block %s", parentID, cb.ID())
+		cb.vm.Ctx.Log.Verbo("could not get parent (ID %s) of block %s", parentID, cb.ID())
```
Comment (StephenButtolph) [inline vms/platformvm/common_blocks.go:167]: This should probably just be removed.

File: networking/voting_handlers.go [hunk 2]
```diff
@@ -340,7 +340,7 @@ func (s *Voting) PullQuery(validatorIDs ids.ShortSet, chainID ids.ID, requestID
 			peers = append(peers, peer)
 			s.log.Verbo("Sending a PullQuery to %s", vID)
 		} else {
-			s.log.Warn("Attempted to send a PullQuery message to a disconnected validator: %s", vID)
+			s.log.Warn("attempted to send a PullQuery message to a disconnected validator: %s", vID)
```
Comment (StephenButtolph) [inline networking/voting_handlers.go:343]: This should be a Debug message (as well as other equivalent logs for different message types)

File: networking/handshake_handlers.go [hunk 3]
```diff
@@ -281,7 +281,7 @@ func (nm *Handshake) Connect(addr salticidae.NetAddr) {
 			return
 		}
 
-		nm.log.Info("Attempting to discover peer at %s", ipStr)
+		nm.log.Debug("Attempting to discover peer at %s", ipStr)
```
Comment (StephenButtolph) [inline networking/handshake_handlers.go:284]: Seems like we are switching to start with a lowercase letter. Here as well?

File: networking/handshake_handlers.go [hunk 4]
```diff
@@ -568,7 +568,7 @@ func (nm *Handshake) checkCompatibility(peerVersion string) bool {
 		// peers major version is too low
 		return false
 	case major > MajorVersion:
-		nm.log.Warn("Peer attempted to connect with a higher major version, this client may need to be updated")
+		nm.log.Debug("peer attempted to connect with a higher major version, this client may need to be updated")
```
Comment (StephenButtolph) [inline networking/handshake_handlers.go:571]: This and the other two version warnings below should be at least `Info messages`, as we want to prompt the user to update.

File: networking/handshake_handlers.go [hunk 5]
```diff
@@ -374,7 +374,7 @@ func (nm *Handshake) SendVersion(peer salticidae.PeerID) error {
 	build := Builder{}
 	v, err := build.Version(nm.networkID, nm.clock.Unix(), toIPDesc(nm.myAddr), ClientVersion)
 	if err != nil {
-		return fmt.Errorf("packing Version failed due to %s", err)
+		return fmt.Errorf("packing version failed due to: %w", err)
```
Comment (StephenButtolph) [inline networking/handshake_handlers.go:377]: It seems like we're keeping the messages capitalized in the other messages, so this should either stay capitalized, or the rest should be made lowercase

**Comment by danlaine** [comment]
@StephenButtolph Addressed the comments

**Pull request merged**